Gulpfile.ts

```diff
@@ -0,0 +1,47 @@
+import {
+    Build,
+    BuildCheckTest,
+    BuildTest,
+    CheckNode,
+    Clean,
+    Compile,
+    Copy,
+    Coverage,
+    Demo,
+    ESLintCheck,
+    ESLintFix,
+    ExecTask,
+    Lint,
+    LintFix,
+    MDLintCheck,
+    MDLintFix,
+    Test,
+} from "@proxima-oss/gulp-tasks";
+import gulp, { TaskFunctionCallback } from "gulp";
+
+// ***** Base Tasks *****
+gulp.task("build", Build);
+gulp.task("check-node", CheckNode);
```

Comment:
looks like check-node is executing inside of the `gulp-tasks` repo? I'd also be concerned that the other tasks are executing there too? Once build is fixed, please do some thorough regression testing before we consider merging

Reply:
oh shit! It might be because the way the paths are coded!

Reply:
Is this resolved?

Reply:
yessir